packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -31,15 +25,94 @@ import {
   Role,
   ServicePrincipal,
 } from 'aws-cdk-lib/aws-iam';
-import { IFunction, Runtime } from 'aws-cdk-lib/aws-lambda';
-import { GraphQLAPIProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import { IFunction, LayerVersion, Runtime } from 'aws-cdk-lib/aws-lambda';
+import { Construct } from 'constructs';
 import path from 'path';
-import {RDSConnectionSecrets} from '@aws-amplify/graphql-transformer-core';
 
 export type OPERATIONS = 'CREATE' | 'UPDATE' | 'DELETE' | 'GET' | 'LIST' | 'SYNC';
 
 const OPERATION_KEY = '__operation';
 
+const RDSLayerMappingID = 'RDSLayerResourceMapping';
+// TODO: This is temporary state, we need to modify this to a production layer
+export const setRDSLayerMappings = (scope: Construct): CfnMapping => new CfnMapping(
+  scope,
+  RDSLayerMappingID,
+  {
+    mapping: {
```

Comment:
can we generate this mapping via a method call so that it's easier to make changes and maintain? The method can take an array of supported region constant, account id and layerName.

Reply:
The version and account # may be different between regions, I don't see much benefit in moving this to an utility method. Can we revisit when we migrate to production account?